src/chrome/content/rules/Steamrep.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Steamrep.com">
+  <target host="steamrep.com" />
+  <target host="www.steamrep.com" />
+
+  <rule from="^http://(www\.)?steamrep\.com/" to="https://www.steamrep.com/" />
```

Comment:
Looks like this actually doesn't work with the full www.steamrep.com -- maybe remove the www prefix?